nextflow.config

```diff
@@ -39,10 +47,12 @@ params {
   // For 100bp reads, this is -60. Mismatches cost -6, gap opening -5 and gap extension -2
   // So -60 would allow 10 mismatches or ~ 8 x 1-2bp indels
   // Bismark default is 0.2 (L,0,-0.2), Bowtie2 default is 0.6 (L,0,-0.6)
+  bismark_align_cpu_per_multicore = null
+  bismark_align_mem_per_multicore = null
 
   // Boilerplate options
   name = false
-  multiqc_config = "$baseDir/assets/multiqc_config.yaml"
```

Comment:
I suppose this slipped in by accident?

Reply:
No, this is intentional - see https://github.com/nf-core/tools/pull/508